code/modules/clothing/glasses/hud.dm

```diff
@@ -40,21 +40,7 @@
 	hud_type = MOB_HUD_MEDICAL_ADVANCED
 	actions_types = list(/datum/action/item_action/toggle)
 
-
-/obj/item/clothing/glasses/hud/health/attack_self(mob/living/user)
```

Comment:
leave this in, we don't want literally everyone to powergame with medhuds

Reply:
The usability of any item regardless of skill is a staple of TG and to be quite frank will not be changed in this pr, alter the availability of medhuds through mapping if you want it changed.

Reply:
pretty sure "staple of TG" makes every medic fucking useless which we want to avoid doing here

Reply:
Medics still have their access, and on regular TG access is all that differentiates between jobs.

Reply:
yeah but i think avoiding chucklefucks screeching for blantant upgrades is a good thing to have after all